docker-compose.yml

```diff
@@ -2,9 +2,20 @@
 version: '3.1'
 
 services:
-  mongo:
-    image: mongo
+  backend-db:
+    image: kartoza/postgis:15-3.3
     restart: always
     environment:
-      MONGO_INITDB_ROOT_USERNAME: pass
-      MONGO_INITDB_ROOT_PASSWORD: pass
+      - POSTGRES_DB=gis
+      - POSTGRES_USER=docker
+      - POSTGRES_PASS=docker
+    ports:
+      - '5432'
+
+  backend:
+    build:
+      dockerfile: Dockerfile
```

Comment:
From here you could specify the backend Dockerfile and the backend dir could be the context.